packages/docs/playwright/with-metamask.test.ts

```diff
@@ -8,9 +8,19 @@ import {
   addPageDiagnostics,
   MetaMask,
   metamaskChromeArgs as args,
-  waitForPopup
+  waitForPopup,
 } from '@usedapp/playwright'
-import {baseUrl} from './constants'
+import { baseUrl } from './constants'
+import {
+  connectToMetamask,
+  firstSign,
+  GNOSIS_SAFE_URL,
+  initGnosisSafe,
+  secondSign,
+  connectToWalletConnect,
+} from './gnosisSafeUtils'
+
+waitForExpect.defaults.timeout = 90000
```

Comment:
Move to `test-setup.ts` (add if not exists), it's weird to change a default in a random test file because it affects or other test files.